Makefile

```diff
@@ -42,12 +43,11 @@ generate-accounts:
 # Dovecot master accounts
 	@ docker run --rm -e MASTER_USER=masterusername -e MASTER_PASS=masterpassword -t $(NAME) /bin/sh -c 'echo "$$MASTER_USER|$$(doveadm pw -s SHA512-CRYPT -u $$MASTER_USER -p $$MASTER_PASS)"' > test/config/dovecot-masters.cf
 
-tests:
-	@ NAME=$(NAME) ./test/bats/bin/bats --timing test/*.bats
+tests: ALWAYS_RUN
+	@ ./test/bats/bin/bats --timing test/*.bats
 
-.PHONY: ALWAYS_RUN
-test/%.bats: ALWAYS_RUN
-	@ ./test/bats/bin/bats $@
+test/%: ALWAYS_RUN
+	@ ./test/bats/bin/bats --timing $@.bats
```

Comment:
As `make` is a weird build system, `ALWAYS_RUN` is a `.PHONY` target, so it will make the recipe for which it is a requirement run everytime. Otherwise, `make` _could_ somehow not run the recipe? Maybe this is not actually needed? I will have a look.

Reply:
Turns out: not needed. I will remove it.

Reply:
I spotted that too before we came up with the real test failure solution and queued it on my personal todo as a follow up, because it wasn't related to "fix transient test failures".

The PR title/description should be changed .

Reply:
Done in fbe020003da793d4fa8dbd2e0433f95971b2af56